DBZ-7291 Retry flush if failed with retriable exceptions each Dialect. (#5854)
debezium/debezium: documentation/modules/ROOT/pages/connectors/jdbc.adoc

 +
If you include this property in the configuration, do not set the `field.include.list` property.

|[[jdbc-property-flush-max-retries]]<<jdbc-property-flush-max-retries, `+flush.max.retries+`>>
|5

> |Specifies the maximum number of attempts to flush changes to the target database on specific database errors.

Reviewer: I'm not strongly attached to it, but I'm inclined to say "certain" vs. "specific" in the absence of information about exactly which errors might cause the failure.  To my mind, "certain" is more open-ended, whereas "specific" leads me to expect that we enumerate those errors somewhere.

If the number of retries is exhausted, the sink connector will enter a _FAILED_ state.

|[[jdbc-property-flush-retry-delay-ms]]<<jdbc-property-flush-retry-delay-ms, `+flush.retry.delay.ms+`>>
|1000

> |Specifies the number of milliseconds to wait in between flush attempts when a flush operation fails and is scheduled to be retried.

Reviewer: ```suggestion
|Specifies the number of milliseconds that the connector waits to retry a flush operation that failed.
```

Reviewer: I think readers can infer that the value applies only if the operation is scheduled to be retried. If you disagree, we can restore that clause.
